src/components/Signup/SignupBanner.scss

```diff
@@ -1,4 +1,4 @@
-.signup-banner {
+.hacDev-signup-banner {
```

Comment:
I don't think we need a prefix like `hacDev` anymore in this repo. The build system automatically adds a parent selector of `hacDev` to all classes. This would also help in later moving some of these components into the SDK without the need to change prefix again.

Reply:
@vikram-raj I think we won't need the prefix according to this ^^

Reply:
Removed `hacDev` prefix.